test/fixtures/hello-cloud/configmap-runner.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hello-cloud-configmap-runner
+  labels:
+    name: hello-cloud-configmap-runner
+    app: hello-cloud
+data:
```

Comment:
This is not the same thing as the pod template resource. That one is not really open for arbitrary manipulation, you can't create one directly. Instead we have to 'hack' one in via `ConfigMap`s. I'll add information to the readme though.